engine/runtime-compiler/src/main/java/org/enso/compiler/dump/DocsUtils.java

```diff
@@ -0,0 +1,155 @@
+package org.enso.compiler.dump;
+
+import static org.enso.scala.wrapper.ScalaConversions.asJava;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.enso.compiler.core.ConstantsNames;
+import org.enso.compiler.core.IR;
+import org.enso.compiler.core.ir.DefinitionArgument;
+import org.enso.compiler.core.ir.Expression;
+import org.enso.compiler.core.ir.Function.Lambda;
+import org.enso.compiler.core.ir.Name;
+import org.enso.compiler.core.ir.expression.Application;
+import org.enso.compiler.core.ir.module.scope.Definition;
+import org.enso.compiler.core.ir.module.scope.definition.Method;
+import org.enso.compiler.core.ir.type.Set;
+import org.enso.compiler.data.BindingsMap;
+import org.enso.compiler.pass.resolve.MethodDefinitions;
+import org.enso.compiler.pass.resolve.TypeNames$;
+import org.enso.compiler.pass.resolve.TypeSignatures;
+import org.enso.compiler.pass.resolve.TypeSignatures$;
+import org.enso.pkg.QualifiedName;
+
+final class DocsUtils {
+  private static final String ANY = "Standard.Base.Any.Any";
```

Comment:
Isn't this available as a constant already somewhere? I'd like to avoid duplicates like that

Reply:
There are few occurrences and even some definitions:

![ANY](https://github.com/user-attachments/assets/414c7531-0e3c-4940-b236-9f8efbbe156f)

The most suitable one would probably be `BuiltinTypes.FQN_ANY`, but it is not public and the dependency on package `org.enso.compiler.pass.analyse.types` doesn't seem correct either.

Reply:
Yeah, I figured `BuiltinTypes.FQN_ANY` should be the source of truth. Shame that we can't include it as a dependency.